client/src/backoffice/TransactionsPage.tsx

```diff
@@ -0,0 +1,207 @@
+  const [total, setTotal] = useState<number>(0);
+  const [modalOpen, setModalOpen] = useState<boolean>(false);
+  const [selectedTx, setSelectedTx] = useState<null | Transaction>(null);
+  const [isFetchingTx, setIsFetchingTx] = useState<null | boolean>(null);
+  const [transactions, setTransactions] = useState<null | Transaction[]>(null);
+
+  useEffect(() => {
+    fetchTransactions();
+  }, [page]);
+
+  const txStatus = {
+    [TX_STATUS.pending]: clock,
+    [TX_STATUS.failed]: error,
+    [TX_STATUS.passed]: checked,
+  };
+
+  const fetchTransactions = () => {
+    setIsFetchingTx(true);
+    setTransactions(null);
+
+    getTransactions(PAGE_SIZE, page  * PAGE_SIZE)
+      .then(response => {
+        if (!response) return;
+        setTransactions(response.transactions);
+        setTotal(response.total);
+      })
+      .catch(error => console.error(error))
+      .finally(() => setIsFetchingTx(false));
+  };
+
+  const handlePageChange = (obj: PaginateAction) => {
+    setPage(obj.selected);
+  };
+
+  const handleFormSubmit = async (
+    values: GasPriceFormValues,
+    actions: FormikActions<GasPriceFormValues>
+  ) => {
+    if (!selectedTx) return;
+    try {
+      actions.setStatus(null);
+      actions.setSubmitting(true);
+
+      const gasPriceInWEI = convertFromGWEI(values.gasPrice);
+      await pumpTransaction(selectedTx.tx_hash, gasPriceInWEI);
+      fetchTransactions();
+      closeEditModal();
+    } catch (error) {
+      actions.setStatus({ ok: false, msg: `Gas price couldn't be changed` });
+    } finally {
+      actions.setSubmitting(false);
+    }
+  };
+
+  const openEditModal = (transaction: Transaction) => {
+    setModalOpen(true);
+    setSelectedTx(transaction);
+  };
+
+  const closeEditModal = () => {
+    setModalOpen(false);
+    setSelectedTx(null);
+  };
+
+
+  return (
+    <div className={'admin-table'}>
+      <h2>Transactions</h2>
+      <div className={'row table-header'}>
+        <div className={'col-xs-1 center'}>#</div>
+        <div className={'col-xs-3'}>Tx Hash</div>
+        <div className={'col-xs-3'}>Signer</div>
+        <div className={'col-xs-2'}>Operation</div>
+        <div className={'col-xs-1 center'}>Status</div>
+        <div className={'col-xs-2 center'}>Gas Price (GWei)</div>
+      </div>
+      <div className={'admin-table-row'}>
+        {isFetchingTx && <Loading />}
+        {transactions && transactions.map((tx, i) => {
+          return (
+            <div className={`row ${i % 2 === 0 ? 'even' : 'odd'}`} key={tx.id}>
+              <div className={'col-xs-1 center'}>{tx.id}</div>
+              <div className={'col-xs-3'}>
+                <a href={`https://etherscan.io/tx/${tx.tx_hash}`} target={"_blank"}>{reduceAddress(tx.tx_hash)}</a>
+              </div>
+              <div className={'col-xs-3'}>
+                <a href={`https://etherscan.io/address/${tx.signer}`} target={"_blank"}>{reduceAddress(tx.signer)}</a>
```

Comment:
Optionally since we're using lot's of this a small function to create `etherscan` links would be good:

Code example:
```typescript
const ETHERSCAN_URL = 'https://etherscan.io'

const etherscanLinks = {
  tx: (hash) => `https://etherscan.io/tx/${tx.tx_hash}`,
  address: (hash) => `https://etherscan.io/tx/${tx.tx_hash}`,
}
```

In use:
```jsx
<a href={etherscanLinks.tx(tx.tx_hash)} target={"_blank"}>{reduceAddress(tx.tx_hash)}</a>
// ...
<a href={etherscanLinks.address(tx.signer)} target={"_blank"}>{reduceAddress(tx.signer)}</a>
// ...
Modify gas price for tx <a href={etherscanLinks.tx(selectedTx.tx_hash)} target={"_blank"}>{selectedTx.tx_hash}</a>.
```

Reply:
Nice!